src/template/style.css

```diff
@@ -0,0 +1,25 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "proxima-nova-bold";
+  font-weight: bold;
+  src: url("../assets/fonts/proxima-nova-bold.otf") format("truetype");
+}
+
+@font-face {
+  font-family: "sentinel-bold";
+  font-weight: bold;
+  src: url("../assets/fonts/sentinel-bold.otf") format("truetype");
+}
+
+:root {
+  --black: rgb(19, 14, 13);
+  --grey: rgb(244, 243, 243);
+  --presentation-background: rgb(204, 204, 204);
+  --red: rgb(231, 43, 71);
+  --sans-serif: "proxima-nova-bold", sans-serif;
+  --serif: "sentinel-bold", serif;
+  --white: rgb(255, 255, 255);
+}
```

Comment:
I think asking them to write all of the CSS for the page miiiight be too much to ask. Can we at least give them some base classes to work with? Maybe we can take of the layout?